scenes/enumerated/problem_11273/problem_11273.py

```diff
@@ -0,0 +1,246 @@
+from manim import *
+from lib.qarakusiscene import TaskNumberBox
+from .text import *
+
+class Problem11273(Scene):
+    def construct(self):
+        scale = .6
+        task_number = TaskNumberBox(task_number_string)
+        rectangle1 = Rectangle(height=3, width=9).scale(scale)
+        rectangle2 = Rectangle(height=3, width=3).scale(scale).align_to(rectangle1, LEFT)
+        rectangle3 = Rectangle(height=3, width=6).scale(scale).align_to(rectangle1, RIGHT)
+        example1 = Rectangle(height=2, width=10).scale(scale)
+        example2 = Rectangle(height=1, width=11).scale(scale)
+        example1_height_label = MathTex(r"2\text{"+cm+"}").next_to(example1, LEFT)
+        example1_width_label = MathTex(r"10\text{"+cm+"}").next_to(example1, UP)
+        example2_height_label = MathTex(r"1\text{"+cm+"}").next_to(example2, LEFT)
+        example2_width_label = MathTex(r"11\text{"+cm+"}").next_to(example2, UP)
+        perimeter1 = MathTex(r"P=24\text{"+cm+"}").move_to(rectangle1.get_center())
+        perimeter2 = MathTex(r"P=12\text{"+cm+"}")
+        perimeter3 = MathTex(r"P=18\text{"+cm+"}")
+        example1_perimeter = MathTex(r"2+10+2+10=24\text{"+cm+"}").to_edge(DOWN, buff=2).to_edge(LEFT)
+        example2_perimeter = MathTex(r"1+11+1+11=24\text{"+cm+"}").to_edge(DOWN, buff=2).to_edge(LEFT)
+        example1_area = MathTex(r"S=20\text{"+cm+"}^2").move_to(rectangle1.get_center()).scale(1.2)
+        example2_area = MathTex(r"S=11\text{"+cm+"}^2").move_to(rectangle1.get_center()).scale(.85)
+        question = MathTex("S=?").move_to(rectangle1.get_center()).scale(1.2)
+        line = Line(UP*1.2, DOWN*1.2, color=YELLOW).next_to(rectangle2, buff=0)
+        sum_of_2_perimeters = MathTex(r"12+18=30\text{"+cm+"}").to_edge(LEFT).to_edge(DOWN, buff=1.9)
+        nvazox_mas = MathTex(r"30-24=6\text{"+cm+"}").next_to(sum_of_2_perimeters, DOWN).align_to(sum_of_2_perimeters, LEFT)
+        height_label = MathTex(r"6:2=3\text{"+cm+"}").next_to(nvazox_mas, DOWN).align_to(nvazox_mas, LEFT)
+        width_plus_height = MathTex(r"24:2=12\text{"+cm+"}").next_to(sum_of_2_perimeters, buff=5.1)
+        width_label = MathTex(r"12-3=9\text{"+cm+"}").next_to(width_plus_height, DOWN).align_to(width_plus_height, LEFT)
```

Comment:
There was a reason people defined everything at the top of the function, and then started writing the code.
This is a correct way to write code for `C` or assembly, but it confuses people, especially in case of long functions.

Define the object when you're gonna use it.

Don't split the code into `data` and `body` segments (with the extra new line). Instead, split the code by its meaning.
e.g. 
```
task_number_code

some_plot_of_first_rectangle
some_plot_of_second_rectangle
some_text_on_top_of_these

some_changes_on_those
...
```

Split the parts the way you would tell the code, "It does this, then that, then the other". It would make both reviewers, readers, testers and your life much easier. (tester might want to test just a small segment of a code, but will create all the objects without need)

Reply:
հմ, իմ պարագայում օրինակ համ գրելուց համ կարդալուց ինձ զգալի ավելի հարմար ա լինում, որ սաղ անիմացիայի տողերը մի մասում լինեն, իսկ հայտարարումները առանձին։ տեղ-տեղ անհարմարություններ ստեղծում ա, բայց 2 ձև էլ փորձել եմ համ գրելը համ ուրիշինը կարդալը, ու տենց զգալի ավելի հարմար ա էղել։ Բայց հասկանում եմ ու ընդունում եմ էլի, որ թեստ անելուց լիքը անկապ օբյեկտ ես ստեղծում, բան, ու էդքան էլ լավ չի։ + եթե անիմացիան օբյեկտը ստեղծելու անմիջապես տակ ա, էլ օբյեկտը չես գնում կարդաս ինչ ա։ Բայց դրանք հաշվի առնելով ինձ սենց ավելի հարմար ա գալիս, եթե փոփոխականներիս անունները նորմալ եմ գրում, մենակ անիմացիաներով կարդացվում ա ինչ պետք ա

Reply:
ասենք որ լիքը self. ինչ-որ բաներ իրար տակ գրած են, իմ աչքը շատ ավելի արագ ա ընկալում որը որից հետո ա, ինչ ա անում անիմացիան, քան եթե օբյեկտ սարքեմ, անիմացնեմ, օբյեկ սարքեմ, անիմացնեմ

Reply:
ու որ քոմմենթների տողերով էլ առանձնացնում եմ իրարից, լավ հարմար ա գալիս

բայց դե նաև պետք ա հաշվի առնել, որ մանիմ անելը իմ փայթ անելու սկիզբն ա հենց ։դ ու ես հենց սենց ուղղակի շատ եմ սովորել, նենց որ ուրիշ փորձով փորձառու ծրագրավորողի աչքը լրիվ ուրիշ ա տենում քան իմը ։դ

Reply:
օքեյ, հմի դու էս կոդը նորմալ կկարդա՞ս։ Որովհետև ես չեմ կռնա կարդամ, ու պատկերացնեմ ինչ կէնե մոտավոր։

Reply:
յոլա էլի, անիմացիաները կարդում եմ, ինիթները ենթադրում եմ որը ինչ ա, եթե չեմ հասկանում, գնում կարդում եմ